test/expected/build/library.e/dest/test-resources/library/e/Test.js

```diff
@@ -0,0 +1,6 @@
+/*
+UI development toolkit for HTML5 (OpenUI5)
+ * (c) Copyright 2009-xxx SAP SE or an SAP affiliate company.
```

Comment:
do we not replace the year in the copyright?

Reply:
We do, but in the ui5.yaml is no ${currentYear} placeholder, there is literally `2009-xxx`